Yoast/wordpress-seo
# Tests/Unit: use PSR-4

## Context

* Improve dev experience 

## Summary

This PR can be summarized in the following changelog entry:

* Tests: use PSR-4

## Relevant technical choices:

### Tests/Unit: make sure all files are namespaced

... including the test bootstrap.

Includes ensuring that all calls to global functions and global constants are now fully qualified.

### Tests/Unit: use PSR-4

* Ensure the `tests/Unit` directory and the subdirectory names use "ProperCase".
* Removing prefixes from the test class names which are not needed in namespaced classes (`WPSEO/Yoast`) and where relevant, update references in other test files.
* Ensure all test class file names match the name of the test class. (best effort, bit too many files to be 100% sure)
* Change the Composer `autoload-dev` directive to use PSR-4 autoloading.
* Update the `phpunit.xml.dist` file to match.
* Update the `phpcs.xml.dist` file to match.

## Test instructions

### Test instructions for the acceptance test before the PR gets merged
This PR can be acceptance tested by following these steps:

* _N/A_ If the tests pass, we're good.
